storage/range.go

```diff
@@ -192,28 +192,13 @@ func (r *Range) stop() {
 
 // Destroy cleans up all data associated with this range.
 func (r *Range) Destroy() error {
```

Comment:
Theoretically we could avoid some of the overhead here by mapping ranges to RocksDB column families, in which case a range drop would become O(1) (but at some extra cost for range splitting and merging). Likely never going to be worth it.